page-index/page-index.js

```diff
@@ -383,9 +383,9 @@ const pageIndex = [
                         type: ANCHOR,
                         href: "#dropping-data"
                     }, {
-                        Title: "Best Practices",
```

Comment:
Best practices section is still in the document. Should Advanced Topics be added to the index instead of replacing?

Reply:
Good point. Added.